luigi/interface.py

```diff
@@ -181,6 +186,17 @@ class PidLockAlreadyTakenExit(SystemExit):
     pass
 
 
+def _detailed_summary_enabled(response, **kwargs):
```

Comment:
I really don't think you needs this, and `**kwargs` really doesn't look clean. But I think you will resolve this issue when you use an explicit kwarg instead of the `**kwarg` capture.

Reply:
I made this function a *decorator*! Looks much cleaner now. 🚶 Is it alright ?